front/src/core-logic/adapters/SiretGatewayThroughBack/HttpSiretGatewayThroughBack.manual.test.ts

```diff
@@ -39,17 +42,28 @@ const simulated = new SimulatedSiretGatewayThroughBack(0, {
   },
 });
 
-const http = new HttpSiretGatewayThroughBack(
-  createManagedAxiosInstance({ baseURL: "http://localhost:1234" }),
+const axiosInstance = createManagedAxiosInstance({
+  baseURL: "http://localhost:1234",
+});
+
+const createHttpClient = configureHttpClient(
+  createAxiosHandlerCreator(axiosInstance),
+);
+
+const siretGateway = new HttpSiretGatewayThroughBack(
+  createHttpClient<SiretTargets>(siretTargets),
```

Comment:
pas besoin du type normalement, c'est inféré :

```suggestion
  createHttpClient(siretTargets),
```